src/main/java/org/terasology/additionalRails/components/ActivableCartComponent.java

```diff
@@ -0,0 +1,15 @@
+package org.terasology.additionalRails.components;
+
+import org.terasology.entitySystem.Component;
+
+/**
+ * Component which makes the cart able to receive {@link org.terasology.additionalRails.events.CartActivatedEvent}.
+ * Every cart, which want to use the Activator Rail, must contain this component.
+ * @author Aleksander Wójtowicz <anuar2k@outlook.com>
+ */
+public class ActivableCartComponent implements Component {
```

Comment:
Do you really need this since you already send an event to the associated entity?